expo/tests/scheduler/test_add_event.py

```diff
@@ -1,100 +1,331 @@
+        response = self.client.get(url, follow=True)
+        nt.assert_equal(response.status_code, 404)
+
+    def test_good_user_get_success(self):
+        login_as(self.privileged_profile, self)
+        url = reverse(self.view_name,
+                      urlconf="scheduler.urls",
+                      args=["GenericEvent", self.eventitem.eventitem_id])
+        response = self.client.get(url)
+        nt.assert_equal(response.status_code, 200)
+        nt.assert_in(self.eventitem.title,
+                     response.content)
+        nt.assert_in(self.eventitem.description,
+                     response.content)
+        nt.assert_in(str(self.eventitem.duration),
+                     response.content)
+
+    def test_good_user_get_class(self):
+        login_as(self.privileged_profile, self)
+        klass = ClassFactory()
+        url = reverse(self.view_name,
+                      urlconf="scheduler.urls",
+                      args=["Class", klass.eventitem_id])
+        response = self.client.get(url)
+        nt.assert_equal(response.status_code, 200)
+        nt.assert_in(klass.title,
+                     response.content)
+        nt.assert_in(klass.description,
+                     response.content)
+        nt.assert_in("1:00",
+                     response.content)
+        nt.assert_in(str(klass.teacher),
+                     response.content)
+
+    def test_good_user_minimal_post(self):
+        Conference.objects.all().delete()
+        Room.objects.all().delete()
+        context = ClassContext()
+        login_as(self.privileged_profile, self)
+        url = reverse(self.view_name,
+                      urlconf="scheduler.urls",
+                      args=["Class", context.bid.eventitem_id])
+        response = self.client.post(
+            url,
+            data={'event-day': context.days[0].pk,
+                  'event-time': "12:00:00",
+                  'event-location': context.room.pk,
+                  'event-max_volunteer': 3,
+                  'event-title': 'New Title',
+                  'event-description': 'New Description',
+                  },
+            follow=True)
+
+        assert_redirects(response, reverse('event_schedule',
+                                           urlconf='scheduler.urls',
+                                           args=["Class"]))
+
+        nt.assert_not_in('<ul class="errorlist">', response.content)
+        nt.assert_in('Events Information', response.content)
+        sessions = Event.objects.filter(eventitem=context.bid, max_volunteer=3)
+        nt.assert_equal(len(sessions), 1)
+        session = sessions.first()
+        nt.assert_equal(session.starttime,
+                        datetime.combine(context.days[0].day,
+                                         time(12, 0, 0, tzinfo=pytz.utc)))
+        nt.assert_in('New Title', response.content)
+        nt.assert_in(str(context.room), response.content)
+        nt.assert_in('3', response.content)
```

Comment:
For how many of our pages would this assertion fail? Surely the character '3' appears on a large number of our pages. Can we either make this more specific or remove it?